emcc.py

```diff
@@ -1893,7 +1923,7 @@ def include_and_export(name):
                                              '_emscripten_stack_get_base',
                                              '_emscripten_stack_get_end']
 
-    if shared.Settings.USE_PTHREADS:
+    if shared.Settings.USE_PTHREADS or shared.Settings.USE_WASM_WORKERS:
       newargs.append('-pthread')
```

Comment:
* we need to enable targeting shared Memory generation and LLVM atomics (and bulk memory) , without bringing in the full pthreads runtime